### PR TITLE
To-do-list P2p review

### PR DESCRIPTION
**Pull request by melashu**
> This project is created to assess our coding review skill.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const lists = document.querySelectorAll(".list");
+lists.forEach((list) => {
+  list.addEventListener("click", () => {
+    const form = list.children[1];
+    const inputBox = form.children[0];
+    const deleteIcon = form.children[1];
+    const menuIcon = form.children[2];
+    deleteIcon.style.display = "block";
+    menuIcon.style.display = "none";
+    list.style.backgroundColor = "#f2f2f2";
```
Comment (GloireIrenge) [inline src/index.js]: This background color is repeated four times ins your file.
I would suggest you to create one item that will be having this background-color and call it only once.

File: module/UI.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+           <input type="checkbox" class="complete" id="${index + 1}">
+
+           <form class="before"> 
+           <input type='text' id="${index + 1}" value="${
+      task.description
+    }" class='task-editable'/>
+    <i class="fa-solid fa-trash-can  delete-icon" id="${index + 1}"></i> 
+    <i class="fa-solid fa-ellipsis-vertical menu-icon"></i>
+    <input type='hidden' value='${task.description}'/>
+
```
Comment (GloireIrenge) [inline module/UI.js]: On this part, on code is not readable. improving the indentation usage could make it more readable and smart

**Review by GloireIrenge — CHANGES_REQUESTED** [review]
### :arrows_counterclockwise: Changes required
giff
Hello @melashu ,
**Highlights 🥇**
✔️ Following Linters
✔️ Descriptive PR Summary
✔️ Descriptive PR Title

congratulation, you have made a great job but you still have some changes to make on your project to make sure you have respected the Javascript best practice. 😊

Cheers and Happy coding 👏 👏 👏

File: src/index.js (hunk 1 above)
Comment (melashu) [reply src/index.js]: @GloireIrenge  Thank you for your feedback

**Review by GloireIrenge — APPROVED** [review]
### :green_circle: Your project is approved
Hello @melashu ,
Congratulations, your project is perfect and all requirements are well done.

**Pull request merged**